### DISCUSSION TITLE
How to allow multiple aspect ratio when using ABR?

### DISCUSSION THREAD

**Discussion by victorcastro89** [OvenMediaLabs/OvenMediaEngine#1441] [Help]
I need to allow 1080x1920 and 1920x1080 video inputs (FHD on portrait and landscape) and transcode into HD and SD while keeping the original aspect ratio. I have tried many configurations, but I could not achieve this. The last idea I had was to configure one virtual host for portrait and another for landscape. Anyone have faced a similar issue?

**Comment by getroot** [comment]
It is difficult to implement this accurately with the current features of OME. The Transcoding webhooks feature, currently under internal testing, allows you to control the output profile for each stream from your control server. Please try this feature when it is officially released in the future.

Alternatively, we currently only provide BypassIfMatch functionality in conditional transcoding, but we plan to expand it to BypassIfNotMatch, IgnoreIfMatch, IgnoreIfNotMatch, etc. in the future. But this is not a high priority.
https://airensoft.gitbook.io/ovenmediaengine/transcoding#conditional-transcoding
